WordPress/src/main/java/org/wordpress/android/ui/mysite/cards/dynamiccard/DynamicCardsViewModelSlice.kt

```diff
@@ -35,7 +38,10 @@ class DynamicCardsViewModelSlice @Inject constructor() {
     }
```

Comment:
I think we can safely remove the `onMoreMenuClick` function above since this is already handled in the toolbar.
Just adding this as a note and we can handle this in a next PR along with the action handling.

Reply:
Handled in https://github.com/wordpress-mobile/WordPress-Android/pull/19810/commits/44df0da426a96deed644ca5ad1a7d4caaa467436 of https://github.com/wordpress-mobile/WordPress-Android/pull/19810